css/styles.css

```diff
@@ -232,21 +236,25 @@ nav ul li a {
     margin: 0;
     font-size: 1.7rem;
 }
+
 .work-card .description h2 {
```

Comment:
i wonder if this font is totally legible this small?/ maybe try a little letter-spacing?

Reply:
👍